README.md

```diff
@@ -1,8 +1,7 @@
 ## MATLAB
 
-
-| :exclamation:  ** MATLAB versions 2022 and 2023 trigger a segfault and do not work with `MATLAB.jl`. Maintainers are attempting to diagnose the cause and fix the issue. **  |
-|-----------------------------------------|
+| :white_check_mark: ** MATLAB versions 2022 and 2023 now work with `MATLAB.jl`.** |
```

Comment:
I the Matlab updates solved the issue, we shall tell the user to update the Matlab in case of failure. Perhaps the update has an ID or something the user can check.

Reply:
I tried to identify the specific update version from which things started to work again. But I am not able to downgrade update version with my university license (only possible with some very expensive professional one apparently, the joy of commercial software). So I guess the best we can do is to tell users to update to the most recent version (I think it is only possible to update to the most recent one anyway).